src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
-  margin: 0;
+@import './reset';
+@import './utils/mixins';
+
+.wrapper {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  box-sizing: initial;
```

Comment:
why did you add this style?

Reply:
box-sizing: initial; ?

there was no task to pass tests. But I tried to pass)
and noticed that border-box doesn't suit here)